packages/frontend/src/view/components/Row.tsx

```diff
@@ -40,7 +40,7 @@ export function RowLabel({
   return (
     <div
       className={cx(
-        'py-2 pr-2 text-sm font-medium leading-5 text-gray-100 md:w-1/5 md:py-0',
+        'py-2 pr-2 text-sm font-medium leading-5 text-zinc-200 md:w-1/5 md:py-0',
         widthRation,
```

Comment:
I see I made typo back then, could you please fix it?

`widthRation` => `widthRatio`

thx ❗

Reply:
[830c582](https://github.com/l2beat/lz-monitoring/pull/102/commits/830c582877207421c5db1c1bfc4e9c391b20ffde)